<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Road Designer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionControlWidget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="areaToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/New_24x24.png</normaloff>:/MainWindow/Resources/New_24x24.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Open_24x24.png</normaloff>:/MainWindow/Resources/Open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionControlWidget">
   <property name="text">
    <string>Control Widget</string>
   </property>
  </action>
  <action name="actionDeleteEdge">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Delete_24x24.png</normaloff>:/MainWindow/Resources/Delete_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Delete Edge</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Save_24x24.png</normaloff>:/MainWindow/Resources/Save_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Undo_24x24.png</normaloff>:/MainWindow/Resources/Undo_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Cut_24x24.png</normaloff>:/MainWindow/Resources/Cut_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Copy_24x24.png</normaloff>:/MainWindow/Resources/Copy_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Paste_24x24.png</normaloff>:/MainWindow/Resources/Paste_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionOpenToAdd">
   <property name="text">
    <string>Open To Add</string>
   </property>
  </action>
  <action name="actionVoronoi">
   <property name="text">
    <string>Voronoi</string>
   </property>
  </action>
  <action name="actionVoronoiCut">
   <property name="text">
    <string>Voronoi Cut</string>
   </property>
  </action>
  <action name="actionModeBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionModeLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionModeSketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sketch</string>
   </property>
  </action>
  <action name="actionShowArea">
   <property name="text">
    <string>Show Area</string>
   </property>
  </action>
  <action name="actionAreaSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Select_24x24.png</normaloff>:/MainWindow/Resources/Select_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionAreaCreate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Create_24x24.png</normaloff>:/MainWindow/Resources/Create_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionHighwaySketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/SketchH_24x24.png</normaloff>Resources/SketchH_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Highway Sketch</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveRoads">
   <property name="text">
    <string>Save Roads</string>
   </property>
  </action>
  <action name="actionLoad_Highways">
   <property name="text">
    <string>Load Highways</string>
   </property>
  </action>
  <action name="actionSave_Highways">
   <property name="text">
    <string>Save Highways</string>
   </property>
  </action>
  <action name="actionBoulevardSketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/SketchB_24x24.png</normaloff>:/MainWindow/Resources/SketchB_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Boulevard Sketch</string>
   </property>
  </action>
  <action name="actionPropertyWidget">
   <property name="text">
    <string>Property Widget</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Debug_24x24.png</normaloff>:/MainWindow/Resources/Debug_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="action3DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/3DView_24x24.png</normaloff>:/MainWindow/Resources/3DView_24x24.png</iconset>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionTerrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/Terrain_32x32.png</normaloff>:/MainWindow/Resources/Terrain_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Terrain</string>
   </property>
  </action>
  <action name="actionLoadRoads">
   <property name="text">
    <string>Load Roads</string>
   </property>
  </action>
  <action name="actionLoadAreas">
   <property name="text">
    <string>Load Areas</string>
   </property>
  </action>
  <action name="actionSaveAreas">
   <property name="text">
    <string>Save Areas</string>
   </property>
  </action>
  <action name="actionGenerateBlocks">
   <property name="text">
    <string>Generate Blocks</string>
   </property>
  </action>
  <action name="actionGenerateParcels">
   <property name="text">
    <string>Generate Parcels</string>
   </property>
  </action>
  <action name="actionGenerateBuildings">
   <property name="text">
    <string>Generate Buildings</string>
   </property>
  </action>
  <action name="actionAreaDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClearRoads">
   <property name="text">
    <string>Clear Roads</string>
   </property>
  </action>
  <action name="actionPMControlWidget">
   <property name="text">
    <string>PM Control Widget</string>
   </property>
  </action>
  <action name="actionHighways">
   <property name="text">
    <string>Highway</string>
   </property>
  </action>
  <action name="actionBoulevard">
   <property name="text">
    <string>Boulevard</string>
   </property>
  </action>
  <action name="actionAvenue">
   <property name="text">
    <string>Avenue</string>
   </property>
  </action>
  <action name="actionLocal_Street">
   <property name="text">
    <string>Local Street</string>
   </property>
  </action>
  <action name="actionDisplayHighway">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highway</string>
   </property>
  </action>
  <action name="actionDisplayBoulevard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Boulevard</string>
   </property>
  </action>
  <action name="actionDisplayAvenue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Avenue</string>
   </property>
  </action>
  <action name="actionDisplayLocalStreet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Local Street</string>
   </property>
  </action>
  <action name="actionHintLine">
   <property name="text">
    <string>Hint Line</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
